modules/nf-core/samtools/sort/main.nf

```diff
@@ -18,12 +19,11 @@ process SAMTOOLS_SORT {
     tuple val(meta), path("*.csi"),  emit: csi,  optional: true
     path  "versions.yml",            emit: versions
 
-    when:
-    task.ext.when == null || task.ext.when
+    when: task.ext.when
 
     script:
-    def args = task.ext.args ?: ''
-    def prefix = task.ext.prefix ?: "${meta.id}"
+    def args = task.ext.args
+    def prefix = task.ext.prefix
```

Comment:
Do we even need this bit at this point, could we just use task.ext.args directly inside the code?

Reply:
Yeah we could! I've also thought about this and it makes more sense, I'll update this PR after the retreat